js-variables-and-numbers/js-variables-and-numbers_social-media-post/js/index.js

```diff
@@ -13,10 +13,21 @@ console.clear();
 
 // --^-- write your code here --^--
 
+const title = "Hey peoples";
+const text = "Guess what happened to me today!";
+const likes = 4501;
+const userAuthor = "elmo";
+const isReported = true;
+
 /*
 2. Log all variables to the console. Increase the likes by one and log the amount of likes again. Adapt your code from 1.1 if necessary.
```

Comment:
da fehlt der schritt  "Increase the likes by one and log the amount of likes again"